projects/sfl-validation/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@cleo-cic/sfl-validation",
```

Comment:
What is the actual `npm` package name going to be for the validation part? I assume it's not `sfl-validation` anymore., so this probably needs to be changed.

Reply:
We don't have a scope decided yet, but I'll rename it to ngx-json-schema-form for now.